benchmarks/src/main/scala/zio/redis/benchmarks/lists/BlMoveBenchmarks.scala

```diff
@@ -17,14 +17,13 @@
 package zio.redis.benchmarks.lists
 
 import org.openjdk.jmh.annotations._
-import zio.ZIO
-import zio.duration._
 import zio.redis._
 import zio.redis.benchmarks._
+import zio.{ZIO, _}
 
 import java.util.concurrent.TimeUnit
 
-@State(Scope.Thread)
+@State(org.openjdk.jmh.annotations.Scope.Thread)
```

Comment:
Why is this necessary? The import is there.

Reply:
Scope conflicts with zio.Scope, I have imported zio.Scope to hide